atomic_reactor/utils/retries.py

```diff
@@ -71,3 +77,25 @@ def get_retrying_requests_session(client_statuses=HTTP_CLIENT_STATUS_RETRY,
     session.hooks['response'] = [hook_log_error_response_content]
 
     return session
+
+
+@backoff.on_exception(
+    backoff.expo,
+    subprocess.CalledProcessError,
+    factor=SKOPEO_BACKOFF_FACTOR,
+    max_tries=SKOPEO_MAX_RETRIES + 1,  # total tries is N retries + 1 initial attempt
+    jitter=None,  # use deterministic backoff, do not apply random jitter
+)
+def run_skopeo(cmd: List[str]) -> bytes:
```

Comment:
can we make it just general : run_cmd ?
it can be used for any cmd if needed

and in logger.warning you can just write f'{cmd} failed ....

Reply:
Done :+1:
Also renamed the constants